Resources/Prototypes/_SpaceCats/Entities/Clothing/Uniforms/jumpskirts.yml

```
@@ -0,0 +1,10 @@
+- type: entity
+  parent: ClothingUniformJumpskirtCaptain
+  id: ClothingUniformJumpskirtWhiteCaptain
+  name: white captain jumpskirt
+  description: White captain jumpsuit skirt symbolizing that your salary clearly exceeds the norm.
+  components:
+  - type: Sprite
+    sprite: _SpaceCats/Clothing/Uniforms/Jumpskirt/white_captain.rsi
+  - type: Clothing
+    sprite: _SpaceCats/Clothing/Uniforms/Jumpskirt/white_captain.rsi
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)